packages/gestalt/src/SideNavigation.css

```diff
@@ -14,13 +15,39 @@
 
 .fullHeight {
   height: 100%;
-  overflow: scroll;
+  overflow-y: scroll;
+}
+
+.contentWidthTransition {
+  transition: width 0.2s;
```

Comment:
`.2s` is used also in Switch and Masony. I took the value from there to make it consistent.
/cc @hectoid

Reply:
@hectoid eventually we should convert this to time tokens